### ISSUE TITLE
Support Kong 1.0.x

### ISSUE THREAD

**Issue by DonMartin76** [Haufe-Lexware/wicked.haufe.io#160]
The Kong API has changed (again) substantially from 0.14.x to 1.0.x (or 0.15.x).

This requires quite some changes to the Kong Adapter.

TBD whether this will land in wicked 1.0.0, or in a later version.

**Comment by kbhuvanamohan** [comment]
We would be really interested to know when would the Kong 1.0.x version will be inducted into Wicked. Thanks!

**Comment by DonMartin76** [comment]
Looking into it over the next days. There are also lifecycle changes regarding how Kong has to update its database. Not super easy to get this right. And that we want (to get it right).

**Comment by DonMartin76** [comment]
I did an effort to support Kong 1.1.2 in the last couple of days, and it works fairly well, but the following thing is blocking us: https://github.com/Kong/kong/issues/4562

When a client wants to refresh the access token using a refresh token, wicked introspects and checks whether the authenticated user id is still valid; this call is no longer supported in Kong as the DAO was completely rewritten. Let's see whether we can surface this functionality again.

**Comment by rduran408** [comment]
Any update on the effort to support Kong 1.x*?

**Comment by DonMartin76** [comment]
Yes, the Kong issue is still blocking this. I am currently working on circumventing this by also handling the tokens inside wicked's API, but it's not quite finished. Work can be found in the `kong_11x` branches.

**Comment by PingStyle** [comment]
I try to replace Kong to 2.xx & 1.5.1 version,but got message 
"Did not receive anonymous access token"
can you tell me what can i do
thanks

**Comment by PingStyle** [comment]
> Yes, the Kong issue is still blocking this. I am currently working on circumventing this by also handling the tokens inside wicked's API, but it's not quite finished. Work can be found in the `kong_11x` branches.

I try to replace Kong to 2.xx & 1.5.1 version,but got message
"Did not receive anonymous access token"
can you tell me what can i do
thanks
